frontend/kesaseteli/employer/src/pages/login.tsx

```diff
@@ -1,25 +1,57 @@
+import { IconSignin } from 'hds-react';
+import useLogin from 'kesaseteli/employer/hooks/backend/useLogin';
 import { GetStaticProps, NextPage } from 'next';
 import { useRouter } from 'next/router';
+import { useTranslation } from 'next-i18next';
 import * as React from 'react';
 import Container from 'shared/components/container/Container';
 import withoutAuth from 'shared/components/hocs/withoutAuth';
 import Layout from 'shared/components/Layout';
 import useClearQueryParams from 'shared/hooks/useClearQueryParams';
 import getServerSideTranslations from 'shared/i18n/get-server-side-translations';
 
+import { $Notification, $PrimaryButton } from '../components/application/login.sc';
+
 const Login: NextPage = () => {
   useClearQueryParams();
+  const { t } = useTranslation();
   const {
     query: { logout, error, sessionExpired },
   } = useRouter();
+  const login = useLogin();
+
+  const getNotificationLabelKey = (): string => {
+    let notificationLabel = `common:loginPage.infoLabel`
+    if (logout) {
+      notificationLabel = `common:loginPage.logoutLabel`
```

Comment:
nitpick but you could return instantly so no need to use let variable.

Reply:
Also, we could re-use `common:header.loginLabel` and `common:header.logoutLabel`. The name header is misleading although it could be renamed to `common:user.logoutLabel` or something

Reply:
Yea, I'll return instantly :+1: 

logoutLabel is a bit different here:
```
  "loginPage": {
    "logoutLabel": "Olet kirjautunut ulos",
    ...
  },
```
```
  "header": {
    "logoutLabel": "Kirjaudu ulos",
    ...
  },
```

Reply:
I renamed `loginPage.logoutLabel` -> `loginPage.logoutMessageLabel` to make it a bit clearer.